settings.gradle.kts

```diff
@@ -14,6 +14,13 @@ dependencyResolutionManagement {
   repositories {
     mavenCentral()
     mavenLocal()
+    // terracotta repository for jmxmp connector
+    maven {
+      setUrl("https://repo.terracotta.org/maven2")
+      content {
+        includeGroupByRegex("""org\.terracotta.*""")
+      }
+    }
```

Comment:
[for reviewer] moving this here allows to avoid duplication between gatherer and scraper to access this repository.